ffs/minerselector/reptop/reptop.go

```diff
@@ -49,6 +49,9 @@ func (rt *RepTop) GetMiners(n int, f ffs.MinerSelectorFilter) ([]ffs.MinerPropos
 		if f.MaxPrice > 0 && sa.Price > f.MaxPrice {
 			continue
 		}
+		if f.PieceSize < sa.MinPieceSize && f.PieceSize > sa.MaxPieceSize {
+			continue
+		}
```

Comment:
Consider min and max piece size of miners.

Reply:
Think you mean `||`

Reply:
Oops, correct.